node/src/browserstack_local_manager.ts

```diff
@@ -45,3 +46,9 @@ export class BrowserStackLocalManager {
 export function makeBrowserStackLocalManagerFactory(): BrowserStackLocalManager {
   return new BrowserStackLocalManager()
 }
+
+export type LocalIdentifier = string
+
+export function makeLocalIdentifier(config: ConfigOptions): LocalIdentifier {
+  return config.browserStack?.localIdentifier ?? Date.now().toString()
```

Comment:
Is there a reason to use the current timestamp instead of a random string? Date is prone to collisions, though unlikely.

Reply:
I wanted to use the CI build number initially, but that didn't seem a good idea after considering, epoch date was the simplest thing I came up that would allow to have a high chance of being unique, but I am open to suggestions (I know we already create a random number, but I avoid normal randoms as they are pseudo-random)

Reply:
What about `Math.random().toString(36).slice(2)`? The chance of this to collide when several Karma instances start simultaneously is much smaller. BrowserStack Local sessions don't last long, so the chance of long-term colliding is even smaller.

Reply:
we use *force* in the BrowserStack Local args, which would kill any previous instances with the same tunnel (just a BTW). I see Math.random works a bit different as it does not allow a seed, so I think it is more reliable, sure, can do

Reply:
`Math.random` in JS is reliable and used widely, unless the use-case is not cryptography